pkg/telemetry/report.proto

```diff
@@ -4,16 +4,18 @@ package telemetry;
 import "gogoproto/gogo.proto";
 
 message Report {
```

Comment:
It's not a good practice to modify existed field in protobuf. 
Make sure it's compatible with old servers/clients.

Reply:
We didin't use protobuf at all, it was only used to generate the structs definitions, and the actual data send to telemetry server is using `json.Marshal()` with those structs.